test/rulesets/Base/fastmath_able.jl

```diff
@@ -125,8 +125,8 @@ const FASTABLE_AST = quote
 
     @testset "binary functions" begin
         @testset "$f(x, y)" for f in (atan, rem, max, min)
-            test_frule(f, 100rand(), 10rand())
-            test_rrule(f, 100rand(), 10rand())
+            test_frule(f, rand_mod_pair()...)
+            test_rrule(f, rand_mod_pair()...)
```

Comment:
Similar here
I probably should have done that, rather than adding the 100, 10 thing